osu.Game.Rulesets.Catch/Edit/CatchEditorPlayfield.cs

```diff
@@ -21,7 +21,8 @@ protected override void LoadComplete()
             // TODO: honor "hit animation" setting?
             CatcherArea.MovableCatcher.CatchFruitOnPlate = false;
 
-            // TODO: disable hit lighting as well
+            CatcherArea.MovableCatcher.GenerateHitLighting.UnbindBindings();
```

Comment:
This is a *really* hacky way to make this work. I'd much rather doing something similar to `OsuEditorPlayfield` where the drawable is expired or the transforms are removed. While that may also be considered hacky, the current plan is to keep all the hacky logic in these editor-local classes, rather than polluting the gameplay implementations.

Ignoring this `UnbindBinding` being something we really don't want, the way you've exposed `GenerateHitLighting` will likely lead to breakage in the future, ie. if a mod or use case comes up where we want to change it. It will pollute the user's configuration.

Reply:
For some reason I missed this obvious thing (configuration polluted).
I changed it to copying the value at load.
A test was dependent on reacting configuration change <https://github.com/ppy/osu/pull/13930/files#diff-cc3c5a1a159c30ec1b45063a93d6693853467611172e2ee42e8c0686a212b41cL222>. I changed the test as it was bad anyway (affecting other tests).

Reply:
Still not sure about adding a property to do this... is there a reason it can't just be expired or removed locally?

Reply:
`HitExplosion`s (drawable hit lightings) and its container are contained in the `Catcher` in the drawable hierarchy (because hit lighting move as the catcher move). And the drawables are not exposed at all.